README.md

```diff
@@ -1,22 +1,22 @@
 # CovidTracker [data and charts]
-You can find on this repository many charts and data about Coronavirus (COVID-19) disease, especially in Europe. Most of data come from [CSSE](https://systems.jhu.edu) (Johns Hopkins Center for Systems Science and Engineering), but I also add data of my own to have up-to-date data.
+In this repository, you'll find many charts and data about the Coronavirus (COVID-19) disease, especially in Europe. Most of the data comes from the [CSSE](https://systems.jhu.edu) (Johns Hopkins Center for Systems Science and Engineering), but I also add some of my own to keep it up-to-date.
 
-French data come frome [INSEE](https://insee.fr/fr/accueil) (Institut National des Statistiques et des Études Économiques) and [Santé Publique France](https://www.santepubliquefrance.fr).
+French data comes from [INSEE](https://insee.fr/fr/accueil) (Institut National des Statistiques et des Études Économiques) and [Santé Publique France](https://www.santepubliquefrance.fr).
 
-For more information and comments in French, you can follow me on Twitter: [@guillaumerozier](http://twitter.com/guillaumerozier).
+For more information and comments in French, follow me on Twitter: [@guillaumerozier](http://twitter.com/guillaumerozier).
```

Comment:
**please** follow me

Reply:
I'd rather not add this, it feels like redundant fluff